src/web/services/auditTrailService.ts

```diff
@@ -0,0 +1,44 @@
+import axios from 'axios';
+
+import { AuditTrailDTO, AuditTrailEvents } from '../../api/entities/AuditTrail';
+import { mapClientTypeIdsToAdminEnums } from '../../api/services/adminServiceHelpers';
+import { backendError } from '../utils/apiError';
+import { getRoleNamesByIds } from '../utils/apiRoles';
+
+export const GetAuditLogs = async (participantId: number) => {
+  try {
+    const result = await axios.get<AuditTrailDTO[]>(`/participants/${participantId}/auditTrail`);
+    return result.data;
+  } catch (e: unknown) {
+    throw backendError(e, 'Could not get audit trail.');
+  }
+};
+
+export const getPrettyAuditDetails = (eventType: AuditTrailEvents, eventData: unknown) => {
+  const data = eventData as Record<string, unknown>;
+  const outputArray = [];
+  // eslint-disable-next-line guard-for-in
+  for (const key in data) {
```

Comment:
I think we can appease the eslint rule by using [Object.keys()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys):
`for (const key of Object.keys(data)) {`

Reply:
good suggestion. done.